bin/set_allowed_mins.py

```diff
@@ -18,17 +21,22 @@
     with open(os.path.join(CONFIG_PATH, 'submitty.json')) as open_file:
         SUBMITTY_CONFIG = json.load(open_file)
 
-    with open(os.path.join(CONFIG_PATH, 'database.json')) as open_file:
-        DATABASE_CONFIG = json.load(open_file)
+    try:
+        with open(os.path.join(CONFIG_PATH, 'database.json')) as open_file:
+            DATABASE_CONFIG = json.load(open_file)
+    except IOError as e:
+        if e.errno == errno.EACCES:
```

Comment:
Why would other `IOError`s be allowed to happen and the script keeps going? If _any_ exception occurs on reading the script, we need to abort since we cannot load the database details.

Perhaps more appropriate would be:

```python
    except IOError as e:
        if e.errno == errno.EACCES:
            HAS_ERROR = True
        else:
            raise
```

such that we re-raise the exception in other cases and then let the script crash out below? But I'm not well-versed enough on types of `IOError` to know whether or not we should care enough about the possible reasons it gets thrown or just group all `IOError` under an umbrella of "could not open file due to permissions".